react/features/whiteboard/actionTypes.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Disable the whiteboard functionality.
+ * {{
+ *      type: DISABLE_WHITEBOARD
+ * }}
+ */
+export const DISABLE_WHITEBOARD: string = 'DISABLE_WHITEBOARD';
+
+/**
+ * Enable the whiteboard functionality.
+ * {{
+ *      type: ENABLE_WHITEBOARD,
+ *      id,
+ *      participantId,
+ *      collabLink
+ * }}
+ */
+export const ENABLE_WHITEBOARD: string = 'ENABLE_WHITEBOARD';
+
+/**
+ * Toggle the whiteboard functionality.
+ * {{
+ *      type: TOGGLE_WHITEBOARD,
+ *      id,
+ *      participantId
+ * }}
+ */
+export const TOGGLE_WHITEBOARD: string = 'TOGGLE_WHITEBOARD';
```

Comment:
I feel like these 3 actions could be reduced to a single one: `SET_WHITEBOARD_STATUS` which takes a variable amount of parameters, including an `open` or `enabled` flag.